kano/utils/hardware.py

```diff
@@ -177,6 +186,8 @@ def get_rpi_model(revision=None):
             model_name = RPI_2_B_KEY
         elif int(revision, 16) & 0x00FFFFFF == 0x00900092:
             model_name = RPI_ZERO_KEY
+        elif revision.strip() == 'a02082':
```

Comment:
Thanks, I thought that it would be a shortcut but I guess not...

I'm fixing now